app/logic/Calendar/Event.ts

```diff
@@ -518,7 +518,7 @@ export class Event extends Observable {
   }
 
   async saveToServer(): Promise<void> {
-    if (!this.isIncomingMeeting && this.participants.hasItems && this.hasChanged()) {
+    if (!this.isIncomingMeeting && this.outgoingInvitation && this.participants.hasItems && this.hasChanged()) {
```

Comment:
Is this condition correct? It's based on the old condition, and it makes sense to me, but you changed it to a different condition, and I didn't understand why. Please adapt it as *necessary* (but not merely because of preference).

Reply:
Neil confirms that this is still correct, just does unnecessary checks that are done in the function, too.